03_TS_events_promises/task_events/events.ts

```diff
@@ -0,0 +1,51 @@
+import EventEmitter from 'events';
+import fs from 'fs';
+import readline from 'readline';
+import { Calculator } from './calculator';
+
+let calculator = new Calculator;
+
+const eventEmitter = new EventEmitter();
+
+(async function processLineByLine() {
+  try {
+    const rl = readline.createInterface({
+      input: fs.createReadStream('./03_TS_events_promises/task_events/input.txt'),
+      crlfDelay: Infinity
+    });
+
+    rl.on('line', (line) => {
+      let numbers: string[] = line.split(' ')
+      let [firstParamenter, secondParameter, operation] = numbers;
+
+      function myEvent(event: string, method: any): void {
+        eventEmitter.on(event, method)
```

Comment:
Please define events of your `eventEmitter` in a separate file:
```
eventEmitter.on('divide', (calculator: Calculator, firstParameter: number, secondParameter: number) => calculator.divide(firstParameter, secondParameter))
// ...
```
You should have a definite list of events - it equals to the number of methods in the `Calculator` class. Also it will help you avoiding `any` typing.